contrib/externs/angular-1.5.js

```diff
@@ -1795,6 +1795,20 @@ angular.$locale = function() {};
  */
 angular.$locale.prototype.id;
 
+/**
+ * @typedef {{
+ *     CURRENCY_SYM: (string),
+ *     GROUP_SEP: (string),
+ *     DECIMAL_SEP: (string)
+ * }}
+ */
+angular.$locale.NUMBER_FORMATS;
```

Comment:
I think this should be a `@record` instead. We will eventually migrate other `@typedef {{...}}`s in this file to be `@record`s.